server/routes/reporting/standard-report/standard-report.controller.js

```diff
@@ -67,13 +68,64 @@
     const reportType = reportKeys[reportKey];
     const config = {};
     const filter = req.session.reportFilter;
+    const buildStandardTableRows = function(tableData, tableHeadings) {
+      const tableRows = tableData.map(data => tableHeadings.map(header => {
+        let output = tableDataMappers[header.dataType](data[snakeToCamel(header.id)]);
+
+        if (header.id === 'juror_number') {
+          return ({
+            html: `<a href=${
+              app.namedRoutes.build('juror-record.overview.get', {jurorNumber: output})
+            }>${
+              output
+            }</a>`,
+          });
+        }
+
+        if (header.id === 'postcode') {
+          output = output.toUpperCase();
+        }
+
+        return ({
+          text: output ? output : '-',
+        });
+      }));
+
+      return tableRows;
+    };
+    const buildPrintUrl = function() {
```

Comment:
Could we put a new line between these function definitions just go give them a bit of breathing room.